src/ManTradesTable.js

```diff
@@ -0,0 +1,178 @@
+
+
+ 
+        
+        const total_weight = new_trades.reduce((sum, t) => {
+            if ( !isNaN(t.weight) && t.weight != '') {
+                console.log(t)
+                return parseFloat(t.weight) + sum
+            } else{
+                return sum
+            }
+        }, 0)
+
+        console.log('total weight', total_weight)
+
+
+        this.setState({trades: new_trades, 
+                        total_weight, total_weight})
+
+
+
+
+      }
+
+    render(){
+
+        const pallete = ['#0074D9', '#FF4136', '#2ECC40', '#FF851B', '#7FDBFF', '#B10DC9', '#FFDC00', '#001f3f', '#39CCCC', '#01FF70', '#85144b', '#F012BE', '#3D9970', '#111111', '#AAAAAA']
+        var data = this.state.trades.filter((t) => (!t.placeholder && !isNaN(t.weight))).map((t, index) => {
+ 
+                const slice =  {title: t.ticker, value : t.weight, color : pallete[index % pallete.length]}
+                console.log(slice)
+                return slice
+            
+        })
+        console.log('data length: ', data.length)
+
+        return (
+        <div>
+            <Typography variant="h4" >
+                Manual Enter Your Holdings to Create Your Portfolio
+            </Typography>
+            <Typography style={{color: "gray"}}>
+                Ensure your percentages add up to 100!
+            </Typography>
+        <div style={{ display: "flex", alignItems: "flex-start", marginTop:20}}>
+            <div style={{ flexGrow: "1",
+                        marginLeft: 20}}>
+            <Table>
+            <TableHead>
+                <TableRow>
+                    <TableCell>Ticker</TableCell>
+                    <TableCell>Percent</TableCell>
+                </TableRow>
+            </TableHead>
+            <TableBody>
+                {
+                    this.state.trades.map((t, index) => {
+                        if (t.placeholder){
+                            return <TradeRow handleEdit={this.handleHoldingEdit} place_ticker='AAPL' place_weight='10' index = {index} color={pallete[index % pallete.length]}/>
+                        } else{
+                            return <TradeRow handleEdit={this.handleHoldingEdit} ticker={t.ticker} place_ticker='AAPL' place_weight='10' weight={t.weight.toString()} index={index} color={pallete[index % pallete.length]}/>
+                        }
+                    })
+                }
+
+            </TableBody>
+        </Table>
+        <div style={{
+            textAlign: "right",
+            marginTop: 5
+        }}>Total: <span style={{ color: this.state.total_weight <= 100 ? 'green' : 'red' }}>{this.state.total_weight}% </span> of 100%</div>
+        
+        
+          <Button    
+          disabled={this.state.total_weight != 100}
+          onClick={this.handleCreate}
+          style={{
+            backgroundImage: "linear-gradient(to top right, #A01A7D, #EC4067)",
+            color: "white",
+            opacity: this.state.total_weight == 100 ? 1.0 : 0.5,
```

Comment:
was this line necessary? for some reason I thought the disabled prop handled discoloring as well, but I guess not

Reply:
It doesn't seem to. Maybe because the background image is set.